app/models/document_collection_non_whitehall_link/govuk_url.rb

```diff
@@ -26,31 +26,17 @@ def title
     content_item["title"]
   end
 
-private
+  private
```

Comment:
Is our styleguide to have `private` indented with the class/end or with the methods?

I've seen both enforced in rubocop at various points?